ComponentKit/Core/CKComponentAnimation.mm

```diff
@@ -28,8 +28,8 @@ static CKComponentAnimationHooks hooksForCAAnimation(CKComponent *component, CAA
   CAAnimation *copiedAnimation = [originalAnimation copy];
   return {
     .didRemount = ^(id context){
-      CALayer *layer = component.viewForAnimation.layer;
-      CKCAssertNotNil(layer, @"%@ has no mounted view, so it cannot be animated", [component class]);
+      CALayer *layer = [component.viewForAnimation valueForKeyPath:layerPath];
```

Comment:
it's equivalent, but i think slightly faster, to avoid `valueForKeyPath:` when possible:

  CALayer *layer = layerPath ? [component.viewForAnimation valueForKeyPath:layerPath] : component.viewForAnimation.layer;